infra/terraform/modules/data_access/team_events/requirements.dev.txt

```diff
@@ -0,0 +1,5 @@
+boto3
+
+pytest
+pytest-spec
+pytest-watch
```

Comment:
Done in this commit: https://github.com/ministryofjustice/analytics-platform-ops/pull/65/commits/c344e57b5c3d64cf0a2610a09bb3d036523d1a89